utils/mock.js

```diff
@@ -14,6 +14,13 @@ function mockQuery (schema, mockedSchema, parsedQuery, fixture, saveFixture, glo
   let mockedQuery
   switch (operationType.toLowerCase()) {
     case 'query':
+      // If there are errors defined on the fixture, return them. This should be, the
+      // first validation because if it's going to mock an error is because there should be
+      // an error, so prevent any extra validation and just return the errors.
+      if (fixture && fixture.errors) {
+        const errors = setFixtureError(fixture.errors)
+        return { errors }
```

Comment:
if `errors` is returned here, does not allow partial result data. e.g., here 2 records are not available (`a` and `c` are `null`), still `b` is provided with the corresponding error. 

```
{
  data: [{
    team: {
      members: [{
        id: 'b',
        name: 'Foo Bar'
      }, null, null]
  }],
  errors: [ Error(could not find id 'a'), Error(could not find other id 'c') ]
}
```

Then, I do believe that if `data` exists in the fixture, then it should be returned. May be just move this block to the next return along `return mockedQuery`.

Reply:
@artola Thanks for reviewing it!

Quick question! I had been trying to find a case where GraphQL return partial data and errors,
but all the cases (for me) are returned like this:
```
{
  "errors": [
    {
      "message": "Cannot return null for non-nullable field Activity.isOwner.",
      "locations": [
        {
          "line": 5,
          "column": 7
        }
      ],
      "path": [
        "getActivityByUuid",
        "isOwner"
      ]
    }
  ],
  "data": null
}
```
is there any case where it's returned with partial data?

Reply:
@estrada9166 Returning partial response is in the spec: https://facebook.github.io/graphql/June2018/#sec-Response

A typical case, for example, I would like to see the last 10 stocks that I bought (code, name, qty, price paid), mostly in my DB, but also "actual price" that depends in some external service.
If the external service is down, I still could deliver the data of the DB and `actualprice=null` plus the error to indicate that this field was not properly resolved.
Probably in the UI, I will print all the data and "not available" for actual price. That could be a way better that do not show anything saying we have a big f... problem. ;)

Reply:
Totally valid! Thanks for the response!

@artola  Would you like to work on it?

Reply:
@artola Also, what do you think if we change [this](https://github.com/EasyGraphQL/easygraphql-tester/blob/master/lib/tester.js#L47) for something like this: 
```js
return {
  data: mockQuery(this.schema, this.mockedSchema, parsedQuery, fixture, saveFixture)
}
```
So, the result will be:
```
{
  data: {
    getMe: {...}
  }
}
```
instead of 
```
{
  getMe: {....}
}
```